src/Loader.lua

```diff
@@ -24,21 +25,21 @@ local Loader = class("Loader", {
 })
 
 function Loader:debugLoad ()
+  local Debug_Spritesheet = AssetLoader:getAsset("DEBUG_TILESET_1")
+  local Grass_Hex_Quad = AssetLoader:getAsset("TILE_GRASS_1")
+  local Water_Hex_Quad = AssetLoader:getAsset("TILE_WATER_1")
+  local Arctic_Hex_Quad = AssetLoader:getAsset("TILE_ARCTIC_1")
+  local Space_Hex_Quad = AssetLoader:getAsset("TILE_SPACE_1")
+  local Planet_1_Quad = AssetLoader:getAsset("TILE_PLANET_1")
+  local Planet_2_Quad = AssetLoader:getAsset("TILE_PLANET_2")
+  local City_Quad = AssetLoader:getAsset("CITY_1")
+  local Debug_Ship_Quad = AssetLoader:getAsset("SHIP_1")
+  local Debug_Troop_Quad = AssetLoader:getAsset("TROOP_1")
+  local Debug_Cursor_Quad = AssetLoader:getAsset("CURSOR_1")
+  local Debug_Spaceship_Quad = AssetLoader:getAsset("SPACE_SHIP_1")
+
```

Comment:
I ignored those because I knew I would be refactoring the Loader class soon anyways.